packages/web-client/src/helpers/item.ts

```diff
@@ -0,0 +1,3 @@
+export interface Item {
```

Comment:
Not a fan of the naming because it's so generic, even if its just there to extend existing interfaces